app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -196,7 +195,34 @@ public void onInput(@NonNull MaterialDialog dialog, CharSequence input) {
                         } else {
                             mFilename = input.toString();
 
-                            new CreatingPdf().execute();
+                            mPath = Environment.getExternalStorageDirectory().getAbsolutePath() +
+                                    HomeFragment.this.getString(R.string.pdf_dir);
+
+                            mPath = mPath + mFilename + HomeFragment.this.getString(R.string.pdf_ext);
+
+                            File tempFile = new File(mPath);
+                            if (tempFile.exists()) {
```

Comment:
Add new function in `https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/FileUtils.java` for checking if a file exists & call that here.